<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoCheck</class>
 <widget class="QWidget" name="AutoCheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>225</y>
     <width>261</width>
     <height>45</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit_stop">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>261</width>
     <height>45</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>460</y>
     <width>471</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/icon/images/猫眼_02.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>170</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/icon/images/08.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>105</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>400</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/icon/images/清除.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_background</zorder>
  <zorder>pushButton_save</zorder>
  <zorder>timeEdit</zorder>
  <zorder>timeEdit_stop</zorder>
  <zorder>textEdit</zorder>
  <zorder>label</zorder>
  <zorder>pushButton_exit</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
